OpenLaTeXStudio/Editor/src/main/java/latexstudio/editor/EditorTopComponent.java

```diff
@@ -73,31 +78,34 @@ public EditorTopComponent() {
         setToolTipText(Bundle.HINT_EditorTopComponent());
         putClientProperty(TopComponent.PROP_CLOSING_DISABLED, Boolean.TRUE);
         putClientProperty(TopComponent.PROP_UNDOCKING_DISABLED, Boolean.TRUE);
-
+        
+        fileActions = new FileActions(this);
```

Comment:
This is bad, because 
1. The constructor is not finished and here we gave a not fully build instance to another class
2. Any construcor should not do much work
Think of a Builder.